include/platform/messagepool.h

```diff
@@ -69,7 +69,7 @@ extern "C" {
  * @param[in] aBufferSize          The size in bytes of a Buffer object.
  *
  */
-void otPlatMessagePoolInit(int aMinNumFreeBuffers, size_t aBufferSize);
+void otPlatMessagePoolInit(uint16_t aMinNumFreeBuffers, size_t aBufferSize);
```

Comment:
also update type (`int` -> `uint16_t`) in the introduction of `param[in]` in Line 68 ?